### ISSUE TITLE
PDID bits description

### ISSUE THREAD

**Issue by jeromeDms** [OpenNuvoton/M031BSP#10]
Hi
Where can I find SYS->PDID bits description ?
I searched in all .h without success.
Thanks
Jerome

**Comment by kchuang1** [comment]
Hi,
You can find it in sys_reg.h.
It is described as below.

     * @var SYS_T::PDID
     * Offset: 0x00  Part Device Identification Number Register
     * ---------------------------------------------------------------------------------------------------
     * |Bits    |Field    |Descriptions
     * | :----: | :----:   | :---- |
     * |[31:0]|PDID    |Part Device Identification Number (Read Only)
     * |         |            |This register reflects device part number code. Software can read this register to identify which device is used.

**Comment by jeromeDms** [comment]
There are no bits description here, only [31:0] = PDID.
It does not show how to identify memory size etc.

**Comment by kchuang1** [comment]
Yes, the PDID is only for distinguish different chip part number.
It doesn't contain the information for memory size.
